src/extent.c

```diff
@@ -1323,7 +1317,7 @@ extent_grow_retained(tsdn_t *tsdn, arena_t *arena,
 
 	extent_init(extent, arena, ptr, alloc_size, false, SC_NSIZES,
 	    arena_extent_sn_next(arena), extent_state_active, zeroed,
-	    committed, true);
+	    committed, true, true);
```

Comment:
Let's make this an enum instead of a new bool parameter? I already have trouble reading all these bool params.

Reply:
Done.